css/styles.css

```diff
@@ -128,25 +132,23 @@ section#resources{
 	flex-direction: column;
 	width:100%;
 	justify-content: center;
-}
-
-section#resources{
 	background-color: #456F74;
 }
+
 section#resources h2{
 	align-self: center;
-	padding-top: 50px;
+	padding-top: 40px;
+	padding-bottom: 20px
 
 }
 
 
 div.resourceleft ul,
 div.resourceright ul{
-	display:flex;
-	flex-direction: row;
+	
```

Comment:
Watch your consistency in line-spacing throughout your CSS. There is no gospel rule for spacing in CSS, but more for readability + consistency for future teammates working with your code :)